src/github-reader.ts

```diff
@@ -42,7 +41,7 @@ export function makeGithubReader(def: GithubDef, userAgent: string): ProjectRead
 class GithubProjectReader implements ProjectReader {
     client: Octokit
     def: GithubDef
-    cache: Map<string, Octokit.ReposGetContentsResponse>
+    cache: Map<string, any>
```

Comment:
Can you explain why we need to relax the type constraint here? If we do need to do this - should always use "unknown" not "any".

Reply:
I have bumped the version further and have narrowed down the types